src/include/distributed/multi_client_executor.h

```diff
@@ -72,6 +75,28 @@ typedef enum
 } BatchQueryStatus;
 
 
+/* Enumeration to track what the task associated with a connection is waiting for */
+typedef enum WaitStatus
+{
+	WAIT_BUSY, /* no IO required, ready to be processed further */
```

Comment:
Not a fan either, but like I said, let's change them in one fell swoop and not by introducing inconsistencies until the original style "ages out".